### DISCUSSION TITLE
¡Ayuda! Se mueven mis elementos al hacer más chica mi pantalla.

### DISCUSSION THREAD

**Discussion by JossPerezT** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#491] [Q&A] [answered]
Hola explorers! 
He estado diseñando mi pokedex pero al abrir mi consola se mueven los demás elementos del pokedex (la imagen de la pokebola). No sé si eso ya es por diseño responsive o si hay alguna etiqueta para que se muevan todos juntos. 
Así se ve en la pantalla normal:
![pokedex 1](https://user-images.githubusercontent.com/84040594/159604798-3f56250b-2129-4287-bcf9-a4921e0a7290.PNG)
Así cuando abro mi consola y se hace pequeña la pantalla: 
![pokedex 2](https://user-images.githubusercontent.com/84040594/159604842-be1b5392-7696-423a-a24e-676fd5861e9e.PNG)
 Aquí esta mi código HTML: 
![HTML](https://user-images.githubusercontent.com/84040594/159604954-0e44de76-e82a-4031-a77c-7040a0d71336.PNG)

Y así el CSS:
![css](https://user-images.githubusercontent.com/84040594/159604982-3cc540a4-5f7e-4f8b-9fe5-c29808751171.PNG)

Ojalá me puedan ayudar, a lo mejor es algo muy básico pero he intentado investigar pero sigue sin salirme.

**Comment by OscarPedroza** [comment, marked as answer]
Hola compañera, por lo que observo el contenedor de la imagen del pokedex ocupa el 100% de la pantalla, y la imagen pokebola se esta posicionando con respecto a ese contendor al hacer más pequeña la pantalla cambiará su posición, ya que esta  como porcentaje, lo que se se me ocurre es crear un segundo contenedor dentro con un ancho del tamaño de la imagen del pokedex y posicionar los elementos dentro de ese nuevo contenedor y asi al cambiar el ancho de la pantalla ya no dependerán de su porcentaje si no del espacio del contenido de la imagen del pokedex.

**Reply by JossPerezT** [reply]
Gracias Oscar, me ayudó mucho y se resolvió el problema que tenía. Ahora todo el pokédex se mueve y cambia su tamaño junto.

**Comment by doguedogue** [comment]
Hay que adaptarla a tu imagen, tus mejores aliados:
```
min-width, min-height, max-width, max-height
position: absolute ,relative y fixed
```

**Reply by JossPerezT** [reply]
Vale. Ocupo más absolute y relative, aún no entiendo muy bien el fixed pero lo implementaré para ver cómo funciona 😁
